__tests__/app.test.js

```diff
@@ -103,3 +102,47 @@ describe('GET /api', () => {
         })
     });
 });
+
+describe('GET /api/reviews/:review_id/comments', () => {
+    it('should respond with all the comments asociated from that particular review  ', () => {
+        return request(app).get('/api/reviews/2/comments')
+        .expect(200)
+        .then((result) => {
+            const resultArr = result.body.comments
```

Comment:
Can we check the comments are sorted as per the ticket, jest-sorted may come in handy.